clients/privacy-center/features/consent/GpcMessages.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Badge,
+  Box,
+  Link,
+  Stack,
+  Text,
+  HStack,
+  WarningTwoIcon,
+} from "@fidesui/react";
+import { GpcStatus } from "./types";
+
+const BADGE_COLORS = {
+  [GpcStatus.NONE]: undefined,
+  [GpcStatus.APPLIED]: "green",
+  [GpcStatus.OVERRIDDEN]: "red",
+};
+
+export const GpcBadge = ({ status }: { status: GpcStatus }) =>
```

Comment:
I see now that this type of banner is being used in multiple designs, including [this](https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/privacy-requests/configuration/ConfigurationNotificationBanner.tsx) notification I used for configuration here. Should this be extracted to our UI library?

Reply:
I think you mean [this line](https://github.com/ethyca/fides/pull/2596/files/7ca3b9efe7d4fd0c1f820291126f490929c67a2c#diff-096eb04be34fc44c29c2de204cc8d13548e2d687a3190c227c35da8ac31fe78eR68). Yeah I was wondering if that ended up somewhere. I'll note this in my design review placeholder ticket.